### PR TITLE
Rm vite workbench 2

### PR DESCRIPTION
**Pull request by robmoffat**

### CODE REVIEW

**Comment by mattjamieson** [comment]
Looks good to me, though we should probably back out a few of the changes I made while testing before we merge

**Comment by robmoffat** [comment]
@mattjamieson @kriswest can you guys give this the once-over?

File: website/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "copy-schemas": "cpy ../src/context/schemas/*.schema.json static/schemas/next",
     "copy-appd": "cpy ../src/app-directory/specification/appd.yaml static/schemas/next --rename app-directory.yaml",
     "build-workbench": "cd ../toolbox/fdc3-workbench && npm install && npm run build",
-    "copy-workbench": "del-cli static/toolbox/fdc3-workbench && npm run build-workbench && cd ../toolbox/fdc3-workbench/build && cpy ** ../../../website/static/toolbox/fdc3-workbench --parents",
+    "copy-workbench": "del-cli static/toolbox/fdc3-workbench && npm run build-workbench && cd ../toolbox/fdc3-workbench/build && cpy ** ../../../website/static/toolbox/fdc3-workbench --parents && find ../../../website/static/toolbox/fdc3-workbench",
```
Comment (mattjamieson) [inline website/package.json:12]:
```suggestion
    "copy-workbench": "del-cli static/toolbox/fdc3-workbench && npm run build-workbench && cd ../toolbox/fdc3-workbench/build && cpy ** ../../../website/static/toolbox/fdc3-workbench --parents",
```

File: toolbox/fdc3-workbench/vite.config.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import react from "@vitejs/plugin-react";
+import { defineConfig } from "vite";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+	base: "/toolbox/fdc3-workbench",
+	build: {
+		outDir: "build",
+		assetsDir: "static",
+		assetsInlineLimit: 0,
```
Comment (mattjamieson) [inline toolbox/fdc3-workbench/vite.config.ts:11]:
```suggestion
```

**Review by mattjamieson — CHANGES_REQUESTED** [review]
LGTM but I've added suggestions to remove the debugging/trial stuff I added while trying to get the netlify build to work. Assuming it doesn't cause issues it would be better to keep the configs as simple as possible for future maintenance!

**Review by mattjamieson — APPROVED** [review]

**Pull request merged**